var/spack/repos/builtin/packages/apple-glu/package.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class AppleGlu(Package):
+    """Placeholder package for Apple GLU"""
+
+    homepage = ""
+
+    maintainers = ["aphecetche"]
+
+    has_code = False
+
+    version("4.0.0")
+
+    provides("glu")
```

Comment:
Should probably be...

```suggestion
    provides("glu", when=sys.platform == "darwin")
```

...as in other package ?

Reply:
indeed.
I should also specify the version (glu@1.3) I guess. 
will change.